scripts/pack_docker.sh

```diff
@@ -12,7 +12,7 @@ set +e
 
 >&2 echo -e "${BGreen}Building docker image${NC}"
 set -e
->&2 sudo DOCKER_BUILDKIT=1 docker build -t skyplane .
+>&2 sudo DOCKER_BUILDKIT=1 docker build -t skyplane --platform linux/x86_64 .
```

Comment:
Is this for building cross platform images on M1? pretty neat!

Reply:
Yeah! This fixes the error: `The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested`